site/templates/siteComponent.pug

```diff
@@ -34,59 +63,127 @@ html(lang='en-US').spectrum.spectrum--light.spectrum--medium
+                          svg.spectrum-Icon.spectrum-UIIcon-CheckmarkMedium.spectrum-Menu-checkmark(focusable='false' aria-hidden='true')
+                            use(xlink:href='#spectrum-css-icon-CheckmarkMedium')
+                        li.spectrum-Menu-item(role='option' tabindex='0' value="darkest")
+                          span.spectrum-Menu-itemLabel Darkest
+                          svg.spectrum-Icon.spectrum-UIIcon-CheckmarkMedium.spectrum-Menu-checkmark(focusable='false' aria-hidden='true')
+                            use(xlink:href='#spectrum-css-icon-CheckmarkMedium')
+                div.spectrum-CSS-switcherContainer
+                  label.spectrum-FieldLabel.spectrum-FieldLabel--left(for='switcher-scale') Scale
+
+                  .spectrum-Dropdown.spectrum-Dropdown--quiet(id="switcher-scale")
+                    button.spectrum-FieldButton.spectrum-FieldButton--quiet.spectrum-Dropdown-trigger(aria-haspopup="true")
+                      span.spectrum-Dropdown-label Medium
+                      svg.spectrum-Icon.spectrum-UIIcon-ChevronDownMedium.spectrum-Dropdown-icon(focusable="false" aria-hidden="true")
+                        use(xlink:href="#spectrum-css-icon-ChevronDownMedium")
+                    .spectrum-Popover.spectrum-Popover--bottom.spectrum-Dropdown-popover.spectrum-Dropdown-popover--quiet
+                      ul.spectrum-Menu(role='listbox')
+                        li.spectrum-Menu-item.is-selected(role='option' aria-selected='true' tabindex='0' value="medium")
+                          span.spectrum-Menu-itemLabel Medium
+                          svg.spectrum-Icon.spectrum-UIIcon-CheckmarkMedium.spectrum-Menu-checkmark(focusable='false' aria-hidden='true')
+                            use(xlink:href='#spectrum-css-icon-CheckmarkMedium')
+                        li.spectrum-Menu-item(role='option' tabindex='0' value="large")
+                          span.spectrum-Menu-itemLabel Large
+                          svg.spectrum-Icon.spectrum-UIIcon-CheckmarkMedium.spectrum-Menu-checkmark(focusable='false' aria-hidden='true')
+                            use(xlink:href='#spectrum-css-icon-CheckmarkMedium')
+
+            table.spectrum-CSSComponent-detailsTable
+              tr
+                th.spectrum-Body--secondary Component status
+                td
+                  div(class=`spectrum-StatusLight spectrum-CSSComponent-status spectrum-StatusLight--${util.getStatusLightVariant(component.dnaStatus)}`)= status
+              tr
+                th.spectrum-Body--secondary Last released
+                td= releaseDate
+              tr
+                th.spectrum-Body--secondary Current version
+                td #{pkg.name}@#{pkg.version}
+            if component.details
+              tr
+                th Details
+                tr!= util.markdown.toHTML(component.details)
+
+            div.spectrum-CSSComponent-resources
+              //-
```

Comment:
No, we should leave it. It will be useful when we have the slugs for each component in the metadata. It will not be rendered in the output.

The same thing is happening in `nav.pug` and `footer.pug`, they have commented out bits that we can re-enable later when we have content.